main/src/main/java/org/mobilitydata/gtfsvalidator/table/GtfsStopTimeSchema.java

```diff
@@ -53,6 +54,7 @@
     @SequenceKey
     int stopSequence();
 
+    @CachedField
```

Comment:
Absolutely! `zone_id` is already cached. I also added `@CachedField` annotation to `translations.txt`.

`java.util.Locale.forLanguageTag()` already has its own cache (I read the implementation and also wrote a unit test) but we can have add our explicit cache just to be sure. At least, the documentation for this function does not say anything about caching, so it may be implementation-dependent.

We keep timezones as `java.util.TimeZone` which does not implement `equals()` and `hashCode()` methods. However, when I call `TimeZone.getTimeZone()` function, it actually returns an instance of `sun.util.calendar.ZoneInfo` which is a subclass of `java.util.TimeZone` (!) and which *does* implement `equals()` and `hashCode()` (!!). That means that I actually can place it to the cache... but only for the Java Runtime that I use. So, a portable solution would be to introduce our own wrapper for TimeZone. Anyway, timezones are used for stops and we do not have many of them, so caching is not vital here.

Reply:
@aababilov 
Shall that be an occasion to explore libraries for time representations? I have good memories of using this
https://www.joda.org/joda-time/#:~:text=Joda%2DTime%20is%20the%20de,business%2Dfriendly%20Apache%202.0%20licence.

Reply:
Google style guide says:

JodaTime has [design flaws](https://blog.joda.org/2009/11/why-jsr-310-isn-joda-time_4941.html) and is no longer in active development except to keep timezone data up to date. Use JavaTime instead.

Reply:
Good to know, thks!

Reply:
You are welcome! Google guide is so strict :(